input/pagecontent/general-requirements.md

```diff
@@ -315,28 +145,39 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
 
 ##### Must Support - Complex Elements
 Complex elements are composed of primitive and/or other complex elements. Elements may have additional rules defined in the profile that also apply, e.g. terminology binding, or invariants. 
 
-For any complex element labelled as *Must Support*, an AU Core Responder **SHALL** be capable of providing at least one of the sub-element values. For some complex types a valid value can be constructed by populating only one sub-element, but usually more than one sub-element is needed. An AU Core Requestor **SHALL** be capable of processing the resource with all sub-elements.
+If a complex element is labelled as *Must Support*, an:
+- AU Core Responders **SHALL** correctly populate the element with at least one of the sub-element values.
+- AU Core Requesters **SHALL** accept resources without error if the element is present and containing any valid sub-elements.
 
 For example, the AU Core MedicationRequest Profile `note` element is labelled *Must Support* and has no *Must Support* sub-elements. When claiming conformance to this profile:
-- AU Core Responders **SHALL** be capable of providing a value in a `MedicationRequest.note` sub-element e.g. `MedicationRequest.note.text`.
-- AU Core Requestors **SHALL** be capable of processing the MedicationRequest resource with a value in `MedicationRequest.note`.
+- AU Core Responders **SHALL** correctly populate a value in any valid `MedicationRequest.note` sub-element if a value is known e.g. `MedicationRequest.note.text`.
+- AU Core Requesters **SHALL** accept the MedicationRequest resource without error if `MedicationRequest.note` is present and containing any valid sub-elements.
 
-For example, if AU Core Patient Profile name element is labelled *Must Support* and has *Must Support* sub-elements "family” and “given”. When claiming conformance to this profile:
-- AU Core Responders **SHALL** be capable of providing a value in `Patient.name.family` and `Patient.name.given`.
-- AU Core Requestors **SHALL** be capable of processing a Patient resource with a value in `Patient.name.family` and `Patient.name.given`.
+If a sub-element is labelled as *Must Support*: 
+- AU Core Responders **SHALL** correctly populate the element with all _Must Support_ sub-elements for which a value is known. 
+- AU Core Requesters **SHALL** accept resource without error if _Must Support_ sub-elements are present and containing any valid value.
+
+For example, in the AU Core Patient Profile, the ``name` element is labelled *Must Support* and has *Must Support* sub-elements `family` and `given`. When claiming conformance to this profile:
+- AU Core Responders **SHALL** correctly populate a value in `Patient.name.family` and `Patient.name.given` if the value for those sub-elements is known.
+- AU Core Requesters **SHALL** accept a Patient resource without error if `Patient.name` present and contains valid values in `family` and `given` sub-element.
```

Comment:
Missing word - AU Core Requesters **SHALL** accept a Patient resource without error if `Patient.name` **_+is+_** present and contains valid values in `family` and `given` sub-element.

Reply:
fixed. Also changed to Practitioner instead of Patient since Practitioner only supports those two sub-elements whereas Patient supports more.